config-connector/solutions/sql/kpt/mysql-ha/README.md

```diff
@@ -0,0 +1,37 @@
+MySQL High Availability
+==================================================
+# NAME
+  mysql-ha
+# SYNOPSIS
+  Config Connector compatible YAMLs for creating a high availability MySQL cluster
+# CONSUMPTION
+  Download the package using [kpt](https://googlecontainertools.github.io/kpt/).
+  ```
+  kpt pkg get https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/config-connector/solutions/sql/kpt/mysql-ha mysql-ha
+  ```
+# REQUIREMENTS
+  A working Config Connector installation managing a project with Cloud SQL Admin API enabled.
```

Comment:
A working `cluster with Config Connector installed and configured to manage` a project with Cloud SQL Admin API enabled.

Reply:
Done